app-crypt/libu2f_host/patches/libu2f_host-1.1.10.patchset

```diff
@@ -0,0 +1,24 @@
+From 2f7fb88111d3e92f9aa9bf0456ac28a20baa4e8b Mon Sep 17 00:00:00 2001
+From: Panagiotis Vasilopoulos <hello@alwayslivid.com>
+Date: Thu, 26 Dec 2019 21:00:00 +0200
+Subject: Disabled help2man
+
+diff --git a/src/Makefile.am b/src/Makefile.am
+index 5dd747a..9cb7dee 100644
+--- a/src/Makefile.am
++++ b/src/Makefile.am
+@@ -31,12 +31,3 @@ cmdline.c cmdline.h: cmdline.ggo Makefile.am
+ 
+ BUILT_SOURCES = cmdline.c cmdline.h
+ MAINTAINERCLEANFILES = $(BUILT_SOURCES)
+-
+-dist_man_MANS = u2f-host.1
+-DISTCLEANFILES = $(dist_man_MANS)
+-
+-u2f-host.1: $(srcdir)/u2f-host.c $(srcdir)/cmdline.ggo $(top_srcdir)/configure.ac | $(builddir)/u2f-host$(EXEEXT)
+-       $(AM_V_GEN)$(HELP2MAN) \
+-               --output=$@ $(builddir)/u2f-host$(EXEEXT) \
+-               --name="Yubico Universal 2nd Factor (U2F) Host Tool" \
+-               --no-info
+-- 
```

Comment:
There is a way to get the man page to build, but you need to supply an option here instead of removing it all